spec/services/admin/participants/search_spec.rb

```diff
@@ -6,7 +6,16 @@
   let(:search) { Admin::Participants::Search }
 
   describe "searching" do
-    let!(:user_1) { create(:user, full_name: "Andrew Armstrong", email: "aaaa@example.com") }
+    let!(:school) { user_1.school }
+
+    let!(:npq_application) { create(:npq_application, user: user_1) }
+
+    let!(:user_1) do
+      create(:user,
+             :induction_coordinator,
```

Comment:
As an induction coordinator they have a school but a different comes up to the `.school` option